src/renderer/react/token-box/containers/token-box.js

```diff
@@ -0,0 +1,17 @@
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import * as localeActions from 'common/locale/actions';
+import { TokenBox } from 'selfkey-ui';
+
+const mapStateToProps = state => {
+	return { locale: state.locale };
```

Comment:
@rodrigopavezi this should be retrieved using a selector function, per the best practices doc I linked.

I know it seems trivial, but if we ever have to refactor the locale reducer you'll be glad you used a selector.